examples/api/test/material/scrollbar/scrollbar.1_test.dart

```diff
@@ -0,0 +1,28 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/scrollbar/scrollbar.1.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('The scrollbar thumb should be visible at all time', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const example.ScrollbarExampleApp(),
+    );
+    await tester.pumpAndSettle(); // Waits for all the paints to be done.
```

Comment:
I noticed I need a `pumpAndSettle()` here, several `pump`s were not doing the trick.

I guess this is because the scrollbar is not painted in the 1st frame as it needs to get the extent of the list. Is there something better to do to make the test pass here?

Reply:
It seems that the following would be ok (Maybe it is clearer than `pumpAndSettle` with a comment such as the one I added).

```dart
    await tester.pump();
    await tester.pump(const Duration(milliseconds: 10)); // Wait for the thumb to start appearing.
```

Reply:
Sure, I made the change in [test: Remove empty line and use pumps](https://github.com/flutter/flutter/pull/151463/commits/627803e735c1ad86a8ce44fc2d0a76dd7077f9da)